src/api/mod.rs

```diff
@@ -23,13 +23,17 @@ pub enum ApiError {
     Database(#[from] crate::db::error::DatabaseError),
     #[error("GitHub error: {0}")]
     Github(#[from] crate::github::GithubError),
+    #[error("GitHub error: {0}")]
+    Upload(#[from] crate::upload::UploadError),
 }
 
 impl<'r, 'o: 'r> Responder<'r, 'o> for ApiError {
     fn respond_to(self, _request: &'r Request<'_>) -> rocket::response::Result<'o> {
+        println!("API error: {}", self);
```

Comment:
should we use `println_error` from the `forc-tracing` crate here so our console output is standardised?